include/nbl/config/BuildConfigOptions.h.in

```diff
@@ -16,6 +16,8 @@
 #cmakedefine _NBL_PLATFORM_OSX_
 #cmakedefine _NBL_PLATFORM_ANDROID_
 
+#cmakedefine _NBL_BUILD_WITH_WAYLAND
```

Comment:
dont even make it optional

Truth is that user does not need to download and set up the wayland display server, but rather its headers, which they can just simply pull off the net.